src/styles.css

```diff
@@ -298,7 +319,8 @@ input {
 }
 
 #recipeName {
-  display: flex; 
+  display: flex;
+  position: relative;
```

Comment:
Curios: Was something not looking right without position: relative?

Reply:
Great question.  In order for an `absolute` positioned element (the image) to be anchored to it's parent, that parent MUST have a `relative` position.

Here's a great read on that, since the mdn docs aren't all that clear for this one
https://css-tricks.com/absolute-positioning-inside-relative-positioning/

Reply:
WOWWW! THANK YOU!